execution/evm/vm.go

```diff
@@ -853,6 +877,8 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 				return nil, firstErr(err, ErrInsufficientBalance)
 			}
 
+			vm.returnData = nil
```

Comment:
So I don't think this will matter in practice, at least not now, but I think strictly we should clear the `returnData` at the beginning of the frame, i.e. right after `CREATE:`. Doing this also makes us consistent with the other implementations.